modules/users/tests/server/user.server.model.tests.js

```diff
@@ -345,36 +345,37 @@ describe('User Model Unit Tests:', function () {
       
     });
 
-    it('should not allow double dotted characters in email address - "abcdef@abc..com"', function (done) {
```

Comment:
I'm confused, you've commented on the outdated code so I'm not sure which part you are referring to exactly.